spec/acceptance/should_create_key_spec.rb

```diff
@@ -101,45 +101,48 @@
         /Registry_value\[32:hklm.Software.Vendor.PuppetLabsTest\w+.SubKeyToPurge.Value3\].ensure: removed/
     ]
 
+    agent_is_x64 = false
+
     windows_agents.each do |agent|
-      keys_created = keys_created_native + (is_x64(agent) ? keys_created_wow : [])
-      values_purged = values_purged_native + (is_x64(agent) ? values_purged_wow : [])
-
-      it 'Registry - Phase 1.a - Create some keys' do
-        apply_manifest_on(agent, phase1, get_apply_opts) do
-          keys_created.each do |key_re|
-            assert_match(key_re, @result.stdout, "Expected #{key_re.inspect} to match the output. (First Run)")
-          end
-          assert_no_match(/err:/, @result.stdout, 'Expected no error messages.')
+      agent_is_x64 = is_x64(agent)
+    end
```

Comment:
Think this `end` can be removed, and the tests can iterate over each node in the nodeset as was done previously... now that `execute_manifest_on(agent,` can be used.